lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -178,28 +178,33 @@ def create_user(
         try:
             with self._connect_to_database() as connection, connection.cursor() as cursor:
                 # Separate roles and privileges from the provided extra user roles.
+                admin_role = False
                 roles = privileges = None
                 if extra_user_roles:
                     extra_user_roles = tuple(extra_user_roles.lower().split(","))
                     cursor.execute(
                         "SELECT rolname FROM pg_roles WHERE rolname IN %s;", (extra_user_roles,)
                     )
-                    roles = [role[0] for role in cursor.fetchall()]
-                    privileges = [
+                    admin_role = "admin" in extra_user_roles
+                    roles = [role[0] for role in cursor.fetchall() if role[0] != "admin"]
+                    privileges = {
                         extra_user_role
                         for extra_user_role in extra_user_roles
-                        if extra_user_role not in roles
-                    ]
+                        if extra_user_role not in roles and extra_user_role != "admin"
+                    }
+                    if "SUPERUSER" in map(str.upper, privileges):
```

Comment:
I think PGB is using superuser. We should change that to admin after this change.

Reply:
Good point. I reverted this part of the code as I realised that Landscape needs a superuser (for plpython extension, a requirement that may or may not be removed at some time, but also due to some schema changes that may happen in a landscape server upgrade).

Reply:
1) we should document this properly. JFMI, which extra_user_role used by pgbouncer?
2) re: `plpython`. We will have to hide it for now (c) Mohamed. Security concerns. This is why it was missing on mid-sync presentation. Mohamed will create a ticket to discuss and hide in documentation for now. No rush necessary today!

Reply:
PgBouncer currently uses `SUPERUSER` (to be able to create other superusers).